src/fsharp/FSComp.txt

```diff
@@ -13,7 +13,7 @@ undefinedNameTypeIn,"The type '%s' is not defined in '%s'."
 undefinedNameRecordLabelOrNamespace,"The record label or namespace '%s' is not defined."
 undefinedNameRecordLabel,"The record label '%s' is not defined."
 undefinedNameSuggestionsIntro,"Maybe you want one of the following:"
-undefinedNameTypeParameter,"The type parameter '%s is not defined."
+undefinedNameTypeParameter,"The type parameter %s is not defined."
```

Comment:
Yes. We add the tick back at call site. This is needed for suggestions to work in the VS quick fix tool